site/content/docs/main/locations.md

```diff
@@ -166,6 +171,40 @@ During backup creation:
 velero backup create full-cluster-backup
 ```
 
+### Create a storage location that uses different credentials
```

Comment:
I have a few questions for this section

1. If you create a Backup storage location without specifying credentials in this way, what credentials are used? Or is it required now to always set different creds when creating a backup storage location? 
2. Can you use the secret for more than one BSL? Would you not want to do that?

Maybe think about making the title more specific? right not it seems a little vague to me? maybe something more like "Create a BackupStorageLocation with unique credentials"?

Reply:
If no credentials are specified, it will default to using the ones provided at install time. We have an example for that use case above so I've linked to it here and added more detail about what is happening in both of those cases. I've also linked to this new section from the existing example. You can use the same credential for more that one BSL, for example if you want to store to multiple regions but it's in the same account, as in the example earlier on this page. I've also updated the title as you suggested :)